### PR TITLE
Brunostar

### PR DESCRIPTION
**Pull request by Brunostar**

### CODE REVIEW

**Comment by elroykanye** [comment]
Please give a pull request title that describes your changes. It will be nice to have look back into the past some time to code like this.

**Review by elroykanye — CHANGES_REQUESTED** [review]
Checkout the comments below

**Review by elroykanye — CHANGES_REQUESTED** [review]
Please give a pull request title that describes your changes. It will be nice to have look back into the past some time to code like this.

File: rc-application/src/main/java/com/transkript/reportcard/business/service/impl/SectionServiceImpl.java [hunk 1]
```diff
@@ -1,10 +1,79 @@
...
+        }));
+    }
+
+    @Override
+    public String updateSection(Long id, SectionDto sectionDto) {
+        if (id != null && sectionRepository.existsById(id)) {
+            School school = schoolService.getSchoolById(sectionDto.getSchoolId());
+            Section section = sectionMapper.mapDtoToSection(sectionDto);
+            section.setId(id);
+            section.setSchool(school);
```
[the remark this answers is not in the document]
Comment (Brunostar) [reply rc-application/src/main/java/com/transkript/reportcard/business/service/impl/SectionServiceImpl.java:70]: Null checking is done in the getSchoolById method

**Pull request merged**